.github/workflows/generate.yml

```diff
@@ -0,0 +1,42 @@
+name: Generate PHP SDK
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  generate-php-sdk:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '18'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Check for 'autorelease' keyword in commit messages
+        run: |
+          if git log --format=%B -n 1 ${{ github.sha }} | grep -q 'autorelease'; then
+            echo "Generating PHP SDK..."
+          else
+            echo "No 'autorelease' keyword found. Skipping PHP SDK generation."
+            exit 0
```

Comment:
This wont stop the workflow i think, it'll continue to the next step (I think? Exit 0 means exit with success) you also cant error out, so there must be a different way